mappings/net/minecraft/predicate/item/EnchantmentPredicate.mapping

```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/class_2035 net/minecraft/predicate/item/EnchantmentPredicate
+	FIELD field_20687 ARRAY_OF_ANY [Lnet/minecraft/class_2035;
```

Comment:
~~Hmm, what is in the array? I need to look at it.~~
The name is good. Just an empty array this is.